docs/getting-started/endpoint-diagnostic-data.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[endpoint-diagnostic-data]]
+= Turn off {endpoint-sec} diagnostic data
+
+By default, {endpoint-sec} streams diagnostic data to your cluster, which Elastic uses to tune protection features. You can stop producing this diagnostic data by configuring the advanced settings in the {endpoint-sec} integration policy.
+
+NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control Kibana telemetry in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
+
+. In the {security-app}, go to *Manage* -> *Endpoints* to view the Endpoints list.
+. On endpoints where you want to disable diagnostic data, click the integration policy in the *Policy* column.
```

Comment:
```suggestion
. Go to the endpoint that you want to disable diagnostic data on, then click the integration policy in the *Policy* column.
```

Reply:
Mainly leaving this to indicate that endpoint should be singular.

Reply:
@nastasha-solomon Gotcha, I think we can stay close to the original and still revise to singular:
```suggestion
. On the endpoint where you want to disable diagnostic data, click the integration policy in the *Policy* column.
```

Reply:
Revision to @nastasha-solomon suggestion: 

> Go to the endpoint for which you want to disable diagnostic data, then click the integration policy in the **Policy** column.

Can also use "locate the endpoint" in lieu of "go to".